src/playground/benchmark.js

```diff
@@ -360,11 +369,12 @@ class Frames {
         this.frames = [];
     }
 
-    update (id, selfTime, totalTime) {
+    update (id, arg, selfTime, totalTime, count) {
```

Comment:
It doesn't look like `arg` is being used here

Reply:
It's not used by this method. Frames.update uses the same arguments as RunningStats.update and Opcodes.update. They're all called together in Profiler.onFrame.

They have sequential arguments instead of deconstructing on there own for performance. The prior version of the update methods would be called per block executed. So it could thousands or more blocks per step of the runtime. This change aggregates those so we could change this API now to deconstruct without the same performance concern.